src/Mod/Path/PathScripts/PathDressupZCorrect.py

```diff
@@ -102,6 +102,8 @@ def _loadFile(self, obj, filename):
         try:
             pointlist = []
             for line in f1.readlines():
+                if line == '\n':
```

Comment:
In case you want to also drop the lines which have whitespace:
```
if not line.strip():
    continue
```

Also, it seems that this function never closes the file.